.github/workflows/test-phpstan.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-versions: ['7.4', '8.0']
+        php-versions: ['8.0']
```

Comment:
Please revert this after #4065 and we'll try it again.

Come to think of it, let's add `7.3` as well. As long as we are supporting all three versions they should all pass static analysis.

Reply:
resetted.